basil.js

```diff
@@ -1494,44 +1494,56 @@ pub.doc = function(doc) {
 /**
  * Sets the size of the current document, if arguments are given.
  * If only one argument is given, both the width and the height are set to this value.
+ * Alternatively a string can be given as the only argument to apply an existing page size preset ("A4", "Letter" etc.).
```

Comment:
Okay. At least I found where the ones the user creates are stored.  

    "/Users/$USERNAME/Library/Preferences/Adobe InDesign/Version 12.0/de_DE/Page Sizes"

It would be cool to parse these xml files and give this user a hint what presets he made by himself. I'll investigate a bit further to find them. You can create them by hitting `⌘ ⇧ p`  

![bildschirmfoto 2017-07-18 um 07 35 31](https://user-images.githubusercontent.com/315106/28302592-8eee8d26-6b8e-11e7-9d49-9ec573fd29b8.png)